src/modules/Dashboard/features/Applet/Schedule/Calendar/EventContainerWrapper/EventContainerWrapper.tsx

```diff
@@ -104,16 +99,25 @@ export const EventContainerWrapper = ({
           );
           const largestArrayOfEventsIdsLength = largestArrayOfEventsIds.eventIds.length;
 
-          if (largestArrayOfEventsIdsLength) {
+          if (largestArrayOfEventsIdsLength && timeContent) {
             timeContent.style.minWidth = `${
               largestArrayOfEventsIdsLength * MIN_EVENT_WIDTH_DAY_VIEW +
               largestArrayOfEventsIdsLength * OFFSET_BETWEEN_EVENTS
             }px`;
           }
         }
-      } else {
+      } else if (timeContent) {
         timeContent.style.minWidth = '';
       }
+
+      // adding class names for event wrapper to show/hide event info by its width and height
+      containerEvents.forEach((eventWrapper) => {
+        const event = eventWrapper.querySelector('.rbc-event') as HTMLElement;
```

Comment:
could you please move this selector to shared CONSTANTS since it appearce among few files?